packages/frontend-shared/cypress/e2e/e2ePluginSetup.ts

```diff
@@ -404,7 +404,15 @@ async function makeE2ETasks () {
       }
     },
     async __internal_openProject ({ argv, projectName }: InternalOpenProjectArgs): Promise<ResetOptionsResult> {
-      if (!scaffoldedProjects.has(projectName)) {
+      let projectMatched = false
```

Comment:
Had to re-work this in order to call `cy.openProject` with a directory inside of a project like `<projectName>/packages/foo` to test monorepos

Reply:
Shame we lose the auto completion :(

Let's do it for now, but I wonder if we can find a way to keep the completion 🤔